main.go

```diff
@@ -55,6 +58,11 @@ type Exporter struct {
 	mutex   sync.RWMutex
 }
 
+// GCP service account keys contain the Project ID
+type ServiceAccount struct {
```

Comment:
Can we rename this type `ServiceAccountFile` ?

Reply:
Yup, corrected that.